management/clipper_manager.py

```diff
@@ -85,39 +85,121 @@ class Clipper:
     """
 
     def __init__(self, host, user, key_path):
```

Comment:
Let's make user and keypath optional arguments. If not local, complain if they're not provided.

Make sure to note that in the method documentation above as well.